buildlib/dockers/docker-compose.yml

```diff
@@ -26,41 +35,49 @@ services:
         MOFED_OS: rhel7.6
         CUDA_VERSION: 11.2.0
         OS_VERSION: 7
+        ARCH: ${ARCH}
+
   centos8-mofed5-cuda11:
-    image: centos8-mofed5-cuda11:2
+    image: centos8-mofed5-cuda11:5
     build:
       context: .
       network: host
       dockerfile: centos-release.Dockerfile
       args:
-        MOFED_VERSION: 5.0-1.0.0.0
+        MOFED_VERSION: 5.4-3.6.8.1
         MOFED_OS: rhel8.2
         CUDA_VERSION: 11.4.0
         OS_VERSION: 8
+        ARCH: ${ARCH}
+
   ubuntu18.04-mofed5-cuda11:
-    image: ubuntu18.04-mofed5-cuda11:3
+    image: ubuntu18.04-mofed5-cuda11:5
     build:
       context: .
       network: host
       dockerfile: ubuntu-release.Dockerfile
       args:
-        MOFED_VERSION: 5.0-1.0.0.0
+        MOFED_VERSION: 5.4-3.6.8.1
```

Comment:
why do we have to bump version? because of ARM?
I'm afraid it means we support MLNX_OFED >=5.4 now, instead of >= 5.0

Reply:
Because of ARM. 
The alternative is to keep separate docker-compose.yml files per platform, as suggested before.